### PR TITLE
lldb pretty printers for relay

### PR DESCRIPTION
**Pull request by Leo-arm**
A set of lldb debugger pretty printers that use the relay
PrettyPrinter functionality to display data structures in
the lldb debugger.
Put the dot.lldbinit file in your home directory as .lldbinit.
Update the file to point to the pretty printer script tvm.py
Restart lldb.

### CODE REVIEW

File: python/tvm/contrib/lldb/tvm.py [hunk 1]
```diff
@@ -0,0 +1,201 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
```
Comment (tqchen) [inline python/tvm/contrib/lldb/tvm.py]: Given that this is a separate package and cannot be imported separately, let us put it under apps/lldb

Comment (Leo-arm) [reply python/tvm/contrib/lldb/tvm.py]: Thanks. Files have been moved.

File: apps/lldb/dot.lldbinit.in [hunk 2]
```diff
@@ -0,0 +1,3 @@
+# Put this file in your home directory as .lldbinit
```
Comment (tqchen) [inline apps/lldb/dot.lldbinit.in]: Given that this file also allows comment, please add an ASF header as in the py file

File: apps/lldb/dot.lldbinit.in [hunk 3]
```diff
@@ -0,0 +1,3 @@
...
+# Update the line below to point to the pretty printer script
+command script import ~/bin/lldb/tvm.py
```
Comment (tqchen) [inline apps/lldb/dot.lldbinit.in]: ~/bin/lldb/tvm.py /path/to/tvm/apps/lldb/tvm.py

File: apps/lldb/dot.lldbinit.in (hunk 2 above)
Comment (Leo-arm) [reply apps/lldb/dot.lldbinit.in]: Done

File: apps/lldb/dot.lldbinit.in (hunk 3 above)
Comment (Leo-arm) [reply apps/lldb/dot.lldbinit.in]: Done.

**Comment by tqchen** [comment]
Thanks @Leo-arm !

**Pull request merged**